example/viam_example_app/pubspec.yaml

```diff
@@ -32,6 +32,7 @@ dependencies:
     sdk: flutter
   viam_sdk:
     path: ../../
+  flutter_webrtc: ^0.9.26
```

Comment:
`Drive by`: this was stopping me from building, i think. Although it may have been with how I was trying to build.

Reply:
this shouldn't be needed -- flutter_webrtc is included with he viam-sdk package

Reply:
i'll take another look.

Reply:
yup you're right, that was my mistake.